--- mosquitto-1.3.1/config.mk	2014-03-25 07:52:40.000000000 +0800
+++ mos/config.mk	2014-04-09 19:42:38.613907636 +0800
@@ -68,7 +68,7 @@
 WITH_PYTHON:=yes
 
 # Build with SRV lookup support.
-WITH_SRV:=yes
+#WITH_SRV:=yes
 
 # =============================================================================
 # End of user configuration
@@ -213,6 +213,6 @@
 
 
 INSTALL?=install
-prefix=/usr/local
+prefix=/usr
 mandir=${prefix}/share/man
 localedir=${prefix}/share/locale
